core/parentchain/block-import-queue/src/lib.rs

```diff
@@ -39,23 +39,26 @@ use error::Result;
 use std::vec::Vec;
 
 /// Trait to push parentchain blocks to an import queue.
-pub trait PushToBlockQueue {
-	type BlockType;
-
+pub trait PushToBlockQueue<BlockType> {
```

Comment:
Changed from an associated type to a generic type parameter, following the `Input Type -> Generic Type Parameter` rule.